tests/test_elections.py

```diff
@@ -145,27 +163,36 @@ def test_empty_query(self):
         self.assertEqual(len(results), 0)
 
     def test_elections(self):
-        results = self._results(api.url_for(ElectionView, office='house', cycle=2012, state='NY', district='07'))
+        results = self._results(api.url_for(ElectionView, office='senate', cycle=2012, state='NY'))
         self.assertEqual(len(results), 1)
+        totals = [each for each in self.totals if each.cycle == 2012]
         expected = {
             'candidate_id': self.candidate.candidate_id,
             'candidate_name': self.candidate.name,
             'incumbent_challenge_full': self.candidate.incumbent_challenge_full,
             'party_full': self.candidate.party_full,
             'committee_ids': [each.committee_id for each in self.committees],
-            'total_receipts': sum(each.receipts for each in self.totals),
-            'total_disbursements': sum(each.disbursements for each in self.totals),
-            'cash_on_hand_end_period': sum(each.last_cash_on_hand_end_period for each in self.totals),
-            'document_description': utils.document_description(
-                self.totals[1].last_report_year,
-                self.totals[1].last_report_type_full,
-            ),
-            'pdf_url': utils.report_pdf_url(
-                self.totals[1].last_report_year,
-                self.totals[1].last_beginning_image_number,
-                'F3',
-                'S',
-            ),
+            'total_receipts': sum(each.receipts for each in totals),
+            'total_disbursements': sum(each.disbursements for each in totals),
+            'cash_on_hand_end_period': sum(each.last_cash_on_hand_end_period for each in totals),
+            'won': False,
+        }
+        self.assertEqual(results[0], expected)
+
+    def test_elections_full(self):
```

Comment:
I'm not seeing a test here for when the year parameter and election_full are mismatched. Did I just miss it?